src/models/polar.jl

```diff
@@ -0,0 +1,509 @@
+# Polar formulation
+#
+struct PolarForm{T, VT, AT} <: AbstractFormulation where {T, VT, AT}
+    network::PS.PowerNetwork
+    device::Device
+    x_min::VT
+    x_max::VT
+    u_min::VT
+    u_max::VT
+    # costs
+    costs_coefficients::AT
+    # Constants
+    active_load::VT
```

Comment:
Move active and reactive loads in the format we want (`CuVector` when we are working on the GPU)